### ISSUE TITLE
README.me should say export ARCH=Linux before running build?

### ISSUE THREAD

**Issue by marcmerlin** [im-tomu/fomu-toolchain#23]
By default it says unsupported platform, build.sh doesn't try to run uname -o or anything

If the intent is to set ARCH manually, can you add this in README?

**Comment by umarcor** [comment]
You are correct. ARCH is expected to be set, and there is no feedback about it. In CI, it is set as https://github.com/im-tomu/fomu-toolchain/blob/master/.github/workflows/update.yml#L66-L70. I think it'd be desirable to enhance the error message (https://github.com/im-tomu/fomu-toolchain/blob/master/build.sh#L133-L134). Say `Unrecognized platform ARCH=${ARCH}` or tell it explicitly: "*you should set envvar ARCH*".

In the readme, instead of suggesting users to build the toolchain themselves, IMHO we should update the first sentence of [Usage](https://github.com/im-tomu/fomu-toolchain#usage). Instead of linking to the latest stable release, we should link to nightly releases and suggest using the latest stable as a fallback only.

@marcmerlin, are you up to proposing a PR? I'm not a native english speaker, but I can review/help with technical details.

**Comment by mithro** [comment]
Any reason to not "autodetect" it if not provided?

**Comment by mithro** [comment]
https://stackoverflow.com/questions/714100/os-detecting-makefile

**Comment by marcmerlin** [comment]
Yeah, Tim's answer is kind of what I had in mind. If not, just stating in the README that the user is supposed to set this, would be enough otherwise.
